packages/itwin/grouping-mapping-widget/src/widget/components/customUI/GroupingMappingCustomUI.ts

```diff
@@ -4,10 +4,20 @@
 *--------------------------------------------------------------------------------------------*/
 import type { ReactElement } from "react";
 
-/**
- * Customized UI type definition for grouping mapping widget.
- */
-export interface GroupingMappingCustomUI {
+export enum GroupingMappingCustomUIType {
+  GROUPING = "Grouping",
+  CONTEXT = "Context"
+}
+
+export type GroupingMappingCustomUI = GroupingCustomUI | ContextCustomUI;
+
+/** Custom UI Definitions */
+
+export interface IGroupingMappingCustomUI {
+  /**
+   * Custom UI type: grouping or context
```

Comment:
Documentation / comment sentences should end with periods.

Reply:
What if new type is added to the enum? Will the developer remember to update the documentation? It would be better to link to the enum itself with `see GroupingMappingCustomUIType` or something.